oauth2client/contrib/gce.py

```diff
@@ -85,32 +142,126 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    def __init__(self,
+                 scope=None,
+                 service_account_email='default',
+                 service_account_info=None,
+                 **unused_kwargs):
         """Constructor for AppAssertionCredentials
 
         Args:
-            scope: string or iterable of strings, scope(s) of the credentials
-                   being requested. Using this argument will have no effect on
-                   the actual scopes for tokens requested. These scopes are
-                   set at VM instance creation time and won't change.
+            scope:
+                string or iterable of strings, scope(s) of the credentials
```

Comment:
I'm trying not to break things. The old argument accepted a string or iterable. Given that scope is a totally non-useful arg, I'd be happiest to remove it completely, but it could unnecessarily break user code. Granted it's entire existence I think there has been a warning that fires if you specify a scope, so maybe we can call that sufficient warning to remove it.